docs_src/src/pages/documentation/api_reference/openapi.mdx

```diff
@@ -215,6 +215,72 @@ app.include_router(subrouter)
 
 </CodeGroup>
 
+## Other Specification Params
+
+We support all the params mentioned in the latest OpenAPI specifications (https://swagger.io/specification/). See an example of using request body below.
+
+<CodeGroup title="Request Body">
+
+```python {{ title: 'untyped' }}
+@app.get("/",
+         request_body=RequestBody(
+             description="hiii",
+             required=True,
+             content={
+                 "application/x-www-form-urlencoded": {
+                     "schema": {
+                         "type": "object",
+                         "properties": {
+                             "name": {
+                                 "description": "Updated name of the pet",
+                                 "type": "string"
+                             },
+                             "status": {
+                                 "description": "Updated status of the pet",
+                                 "type": "string"
+                             }
+                         },
+                         "required": ["status"]
+                     }
+                 }
+             }
+         ))
```

Comment:
The user should not be expected to type in openapi json when typing in the request body. it should either be a class or a typed dictionary

Reply:
hi, isn't this what we meant when we decided we "provide" the necessary params  at first & we "support" all params, if the user wants them?

Reply:
@VishnuSanal , would you say the `body` of a request is not a necessary param?